tests/foreman/ui/test_contentview.py

```diff
@@ -1263,104 +823,6 @@ def test_positive_delete_version_without_refresh(self):
                 self.assertIsNotNone(self.content_views.wait_until_element(
                     locators['contentviews.version_name'] % cvv))
 
-    @run_in_one_thread
-    @run_only_on('sat')
-    @tier2
-    def test_positive_promote_CV_with_custom_user_role_and_filters(self):
-        """Publish and promote cv with user with custom role and filter
-
-        :id: a07fe3df-8645-4a0c-8c56-3f8314ae4878
-
-        :expectedresults: CV should be published and promoted successfully
-
-        :CaseLevel: Integration
-        """
-        myrole = get_role_by_bz(1306359)
-        username = gen_string('alpha')
-        user_password = gen_string('alphanumeric')
-        entities.User(
-            login=username,
-            role=[myrole],
-            password=user_password,
-            organization=[self.organization],
-        ).create()
-        repo_name = gen_string('alpha')
-        env_name = gen_string('alpha')
-        cv_name = gen_string('alpha')
-        with Session(self, username, user_password) as session:
-            # Create Life-cycle environment
-            make_lifecycle_environment(session, name=env_name)
-            # Creates a CV along with product and sync'ed repository
-            self.setup_to_create_cv(repo_name=repo_name)
-            # Create content-view
-            make_contentview(session, name=cv_name)
-            self.assertIsNotNone(self.content_views.search(cv_name))
-            # Add repository to selected CV
-            self.content_views.add_remove_repos(cv_name, [repo_name])
-            # Publish and promote CV to next environment
-            self.content_views.publish(cv_name)
-            self.assertIsNotNone(
-                self.content_views.version_search(cv_name, 'Version 1.0'))
-            status = self.content_views.promote(cv_name, 'Version 1', env_name)
-            self.assertIn('Promoted to {}'.format(env_name), status)
-
-    @run_only_on('sat')
-    @tier2
-    def test_positive_remove_renamed_cv_version_from_default_env(self):
```

Comment:
that coverage basically has no value

Reply:
Are you sure? Sounds more like coverage for some specific regression to me.

Reply:
there is no tags that can provide that information. Anyway, they testing lce removal from renamed cv, not even existence, so yeah - that sounds weird

Reply:
As a tester - I just don't feel value from it